packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -745,6 +745,19 @@ Future<void> _buildGradleProjectV2(
     );
   }
 
+  final String exclamationMark = terminal.color('[!]', TerminalColor.red);
+  final bool usesAndroidX = isAppUsingAndroidX(flutterProject.android.hostAppGradleRoot);
+
+  if (usesAndroidX) {
+    BuildEvent('app-using-android-x').send();
+  } else if (!usesAndroidX) {
+    BuildEvent('app-not-using-android-x').send();
+    printStatus('$exclamationMark Your app isn\'t using AndroidX.', emphasis: true);
+    printStatus(
```

Comment:
I think the Flutter style guide says to do this like:
```dart
  printStatus(
    'To avoid potential build failures, you can quickly migrate your app '
    'by following the steps on https://goo.gl/CP92wY.',
    indent: 4,
  );
```